plugins/network-elements/nicira-nvp/src/main/java/com/cloud/network/nicira/ControlClusterStatus.java

```diff
@@ -84,4 +89,17 @@ public int getActiveCount() {
         }
 
     }
+
+    public class ClusterRoleConfig {
```

Comment:
Do those inner classes need to be public?

It's also something we can fix after the merge, unless there is a reason for that.